cmd/snap/cmd_run.go

```diff
@@ -249,5 +261,15 @@ func runSnapConfine(info *snap.Info, securityTag, snapApp, command, hook string,
 	cmd = append(cmd, snapApp)
 	cmd = append(cmd, args...)
 
+	// FIXME2: for now "ForceDevMode" will prevent snap-confine to be
+	//         run on distros like fedora/suse etc. which is ok because
+	//         snap-confine is a non-op on these distros anyway.
```

Comment:
That's not true since snap-confine does a lot more than running apparmor. The comment also doesn't explain _why_ it's doing that, and I can't tell either. As we know, having an out-of-date snap-confine may create all kinds of compatibility issues with the system. This is `snap-wrap`. :)